catalog/MDCDragons/MDCDragonsController.swift

```diff
@@ -356,22 +377,37 @@ extension MDCDragonsController {
 
   func searchBarTextDidBeginEditing(_ searchBar: UISearchBar) {
     isSearchActive = true
-    tableView.reloadData()
+    refreshTable()
   }
 
   @objc func dismissKeyboard() {
     self.view.endEditing(true)
     isSearchActive = false
-    tableView.reloadData()
+
+  }
+
+  func refreshTable() {
+    if #available(iOS 14.0, *) {
+      #if compiler(>=5.3)
```

Comment:
Why the #if compiler check here?

Reply:
This is AFAIK the best way to do conditional compilation in Swift. The `#available` check on the line above gates the functionality at run time, but the compiler will still attempt to build the body of the if statement, and will fail if it encounters undefined symbols.

Reply:
We had to do something similar with the new UIPointerInteraction APIs (see: https://github.com/material-components/material-components-ios/blob/5a2b2caae09680e38950dfb4c81c2c14f7a57359/components/Buttons/examples/ButtonsPointerInteractionExample.swift#L65)

Reply:
Yep! As Bryan said, for earlier Xcodes (pre 12, i.e. 11.5), the older Swift version is installed, which will not include some of the iOS 14 new symbols, like NSDiffableDataSourceSectionSnapshot which will then cause build failures.